package.json

```diff
@@ -88,6 +88,20 @@
         "title": "Copy note"
       }
     ],
+    "keybindings": [
+      {
+        "key": "ctrl+alt+n",
+        "command": "code-annotation.addNote",
+        "mac": "cmd+alt+k",
```

Comment:
I think there's a typo here, and on the other mac keybinding too..

Reply:
Oh... you right. I will fix it in a moment.